src/sections/General/Footer/index.js

```diff
@@ -47,7 +47,7 @@ const Footer = () => {
                   <li>
                     <Link
                       className="category-link"
-                      to="/resources/category/webassembly-filters"
+                      to="/learn/learning-paths"
                     >WebAssembly Filters
```

Comment:
replace text "WebAssembly Filters" to "Learning Paths"

Reply:
Lol, good catch.😂 I really didnt see it.

Reply:
https://layer5io.slack.com/archives/C015QJKUMPU/p1642531158024400 @Abhijay007

Reply:
@YashKamboj the text also needs to be replaced.